mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/grouping_functions.py

```diff
@@ -65,6 +65,7 @@ def group_input_geometries(input_geometries_file, group_size):
         try:
             groups[group_id_string]
         except KeyError:
+            # todo: change this and call feature_geoms -> geojson
```

Comment:
Is this still planned? Or are we going to work on it in a later step?

Reply:
will be dealt with in the near future